src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep2.java

```diff
@@ -31,15 +33,20 @@ public CreateNewWikiPageObjectStep2(WebDriver driver) {
   }
 
   public void selectCategory(int categoryId) {
+    if (wikiCategoryList.size() == 0) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1155)